### ISSUE TITLE
Visiting .user.js URL on a domain that doesn't exist makes TM for Chrome go bonkers

### ISSUE THREAD

**Issue by JasonBarnabe** [Tampermonkey/tampermonkey#2278]
With Tampermonkey for Chrome, visit https://thisdoesnot.exist/test.user.js

## Expected Behavior

Just a normal Chrome error page.

## Actual Behavior

Tampermonkey's "intermediate step" keeps opening and closing forever.

## Specifications

- Chrome 131.0.6778.85
- TM 5.3.2
- Ubuntu 24.04.01

**Comment by derjanb** [comment]
Yeah. I have to think about how to fix this. Tampermonkey detects that there is no script, blocks script detection and hands the URL over to Chrome, and unblocks script detection. Then after some (short) time Chrome retries to load the URL (like it does with all/many error pages) and Tampermonkey intercepts the request again. And in theory this is right, because on every Chrome-triggered reload a script could possibly appear.
